lib/push.js

```diff
@@ -59,6 +71,24 @@ module.exports = function (log, db) {
     },
 
     /**
+     *  Notifies all devices (except the one who joined) that a new device joined the account
+     *
+     * @param uid
+     * @param deviceName
+     * @param currentDeviceId
+     * @promise
+     */
+    notifyNewDevice: function notifyNewDevice(uid, deviceName, currentDeviceId) {
```

Comment:
@eoger if we are going to do https://github.com/mozilla/fxa-auth-server/issues/1139 in the future you might want to rethink the naming here. 

cc @rfk